### ISSUE TITLE
Adding a password in connection

### ISSUE THREAD

**Issue by shahramSo** [oysteinkrog/SQLite.Net-PCL#328]
I like to know that , is it possible to add a password in connection string? I have a already encrypted database. but how to connect with password by Sqlite.Net-Pcl?
Thanks

**Comment by PandaL33** [comment]
Do you solve the problem?

**Comment by shahramSo** [comment]
No, there is no such solution for Sqlite.Net-Pcl except two website as follows but it's not free:
1. SQLCipher.
2. sqlite-crypt.com
